Pring/File.swift

```diff
@@ -195,7 +195,7 @@ public final class File: NSObject {
         self.url = url
     }
 
-    internal convenience init?(property: [String: String]) {
```

Comment:
Why should I expose this initialization method?
I've already prepared `convenience` .

Reply:
This is a way of being used that is not supposed.
What is the reason for having information other than MetaData?